homework/list.js

```diff
@@ -6,17 +6,24 @@
   var _wrapper = document.querySelector('#note-list-wrapper');
 
   function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
+    var li = new list();
```

Comment:
You should **respect your own work**. If you name instances and constructors so arbitrarily like this, no one, include yourself, could maintain it after 3 months, particularly for those projects grow fast. And though it's only a homework, the importance is it would be like a signature shows that you may be a programmer used to write code like this. So, I'm not pissing you, but I really worry about some tiny issue could be ignored until it affects your opportunities in the future.